src/gtfs_structures.rs

```diff
@@ -95,17 +95,19 @@ where
     NaiveDate::parse_from_str(&s, "%Y%m%d").map_err(serde::de::Error::custom)
 }
 
+pub fn parse_time(s: String) -> u16 {
+    let v: Vec<&str> = s.split(':').collect();
+
+    &v[0].parse().expect(&format!("Invalid time format {}", s)) * 60u16
+        + &v[1].parse().expect(&format!("Invalid time format {}", s))
```

Comment:
You should not fail, you should return Err

Reply:
I’ve added a commit to handle that, thanks for the help